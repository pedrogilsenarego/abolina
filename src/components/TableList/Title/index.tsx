import { FunctionComponent } from 'react'
import { Box } from '@mui/material'

interface Props {
  label: string
}

const Title: FunctionComponent<Props> = ({ label }: Props) => (
  <Box
    py="23px"
    justifySelf="flex-start"
    alignSelf="flex-start"
    fontWeight="500"
    fontSize="20px"
    color="#333333"
  >
    {label}
  </Box>
)

export default Title
