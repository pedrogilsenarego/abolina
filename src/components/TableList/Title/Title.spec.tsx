import { render } from '@testing-library/react'
import Title from '.'

describe('Title', () => {
  it('should be render correctly', () => {
    const { container } = render(<Title label="Title" />)

    expect(container).toMatchSnapshot()
  })
})
