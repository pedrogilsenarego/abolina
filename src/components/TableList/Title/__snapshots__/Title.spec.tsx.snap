// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should be render correctly 1`] = `
<div>
  <div
    class="MuiBox-root css-23cccr"
  >
    Title
  </div>
</div>
`;
